toolkit/themes/shared/design-system/tokens-brand.css

```diff
@@ -25,33 +49,7 @@
     --link-color-active: var(--color-accent-primary-active);
     --link-color-visited: var(--link-color);
 
-    /** Color **/
-    --color-accent-primary: light-dark(var(--color-blue-50), var(--color-cyan-50));
-    --color-accent-primary-hover: light-dark(var(--color-blue-60), var(--color-cyan-30));
-    --color-accent-primary-active: light-dark(var(--color-blue-70), var(--color-cyan-20));
-    --color-canvas: light-dark(var(--color-white), var(--color-gray-90));
-
-    /** Font size **/
-    --font-size-root: 15px;       /* Set at the `:root`. Do not use */
-    --font-size-small: 0.867rem;  /* 13px */
-    --font-size-large: 1.133rem;  /* 17px */
-    --font-size-xlarge: 1.467rem; /* 22px */
-    --font-size-xxlarge: 1.6rem;  /* 24px */
-
     /** Text **/
     --text-color: light-dark(var(--color-gray-100), var(--color-gray-05));
   }
 }
-
-@layer tokens-prefers-contrast {
-  @media (prefers-contrast) {
-    :root,
-    :host(.anonymous-content-host) {
-      /* Border */
-      --border-interactive-color: var(--text-color);
-      --border-interactive-color-hover: var(--border-interactive-color);
-      --border-interactive-color-active: var(--border-interactive-color);
-      --border-interactive-color-disabled: var(--border-interactive-color);
-    }
-  }
-}
```

Comment:
I left this out on purpose since it's getting moved/removed in phabricator.services.mozilla.com/D202767